analyzers/tests/SonarAnalyzer.UnitTest/Helpers/ProjectTypeHelperTests.cs

```diff
@@ -18,24 +18,40 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+using System.Collections.Generic;
 using FluentAssertions;
+using Microsoft.CodeAnalysis;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SonarAnalyzer.Helpers;
+using SonarAnalyzer.UnitTest.MetadataReferences;
+using SonarAnalyzer.UnitTest.TestFramework;
 
 namespace SonarAnalyzer.UnitTest.Helpers
 {
     [TestClass]
     public class ProjectTypeHelperTests
```

Comment:
can you also add a small test for `IsTest(this SyntaxNodeAnalysisContext context)` ?

Reply:
As the context has a longer chain, I swapped existing tests to context and added small one for compilation (it produces less scaffolding for more value like this)